woocommerce/woocommerce-ios
# [Tablet products] Show an empty view on initialization

Closes: #11880 


## Why

Right now, the secondary navigation stack is empty before any product form is shown (like from adding or selecting a product). This PR shows an empty view for the initial state.

## How

Show `EmptyStateViewController` similar to the orders tab using the product list empty view image.

## Testing instructions


Hand-testing is optional, as it's straightforward to verify.

Prerequisite: a large device where the split view is supported.

- In code, enable `splitViewInProductsTab` feature flag in `DefaultFeatureFlagService`
- Launch the app
- Log in if needed
- Go to the products tab --> the empty view should be shown in the secondary view

🗒️ there's a known issue where the empty view is shown after the split view collapses (expanded -> collapsed) on tablets

## Screenshots


dark | light
-- | --
<img src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/d15aaddb-1875-4177-b093-e239ddc18592" width="500" /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/aaa354f0-7616-46e4-a40d-8d4f15662c62" width="500" />

---
